generators/app/templates/__tests__/redux/auth/actions.js

```diff
@@ -1,47 +1,33 @@
-import { actionCreators, actions } from '@redux/auth/actions';
+import {actionCreators, actions} from '@redux/auth/actions';
```

Comment:
Undo this change. There is a linter problem in the project that will be solved in #114